operations/mimir-mixin/dashboards/dashboard-queries.libsonnet

```diff
@@ -241,7 +241,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
         sum by (%(groupByCluster)s, %(groupByLabels)s) (%(perIngesterQuery)s)
```

Comment:
[nit] Given you're on it, can you consider adding wrapping the two sides of `or` with parentesis (and then their content intended) just for a bit more clarity when reading the query?

Something like this:

```
(
  # Classic storage
  ...
) or (
  # Ingest storage
  ...
)
```